documentation/dotnet-monitor.md

```diff
@@ -0,0 +1,309 @@
+
+# Now in the context of the docker container
+dotnet tool install -g dotnet-dump
+/root/.dotnet/tools/dotnet-dump analyze /dumpFiles/core_20200618_083349
+```
+
+### GC Dump
+
+The `/gcdump` endpoint returns a GC dump of the target process.
+
+To collect a GC dump, run the following command:
+
+In PowerShell,
+```powershell
+PS> Start-Process http://localhost:52323/gcdump
+```
+
+In bash,
+```bash
+$ wget --content-disposition http://localhost:52323/gcdump
+```
+
+Unlike in the case of process dumps, the resulting GC dump is a portable format and can be analyzed by Visual Studio and [perfview](https://github.com/microsoft/perfview) regardless of the platform it was collected on. To learn more about when to collect GC dumps and how to analyze them, take a look at our [earlier blog post](https://devblogs.microsoft.com/dotnet/collecting-and-analyzing-memory-dumps/).
+
+### Traces
+
+The `/trace` endpoint returns a trace of the target process. The default trace profile include sampled CPU stacks, HTTP request start/stop events, and metrics for a duration of 30 seconds.
+
+The duration of collection can be customized via the `durationSeconds` querystring parameter The diagnostic data can be customized via the `profile` querystring parameter to include any combination of the preset profiles:
+
+- `CPU` (CPU profiler),
+- `Http` (Request start/stop events from ASP.NET Core),
+- `Logs` (Logging from the `EventSourceLogger` from `Microsoft.Extensions.Logging` library); and
+- `Metrics`(Runtime and ASP.NET Core `EventCounters`).
+
+For example, a request to  `/trace?profile=cpu,logs` will enable the collection of just the CPU profiler and logs.
+
+In addition to the `GET` endpoint, there is `POST` version of the endpoint that allows you to specify what `EventSource` providers to enable via the request body. 
+
+To collect a trace of the target process, run the following command:
+
+In PowerShell,
+
+```powershell
+PS> Start-Process http://localhost:52323/trace
+```
+
+In bash,
+```bash
+$ wget --content-disposition http://localhost:52323/trace
+```
+
+The resulting `.nettrace` file can be analyzed with both Visual Studio and PerfView.
+
+### Logs
+
+The `/logs` endpoint will stream logs from the target process for a duration of 30 seconds.
+
+The duration of collection can be customized via the `durationSeconds` querystring parameter. The logs endpoint is capable of returning either newline delimited JSON([application/x-ndjson](https://github.com/ndjson/ndjson-spec)) or the Event stream format([text/event-stream](https://developer.mozilla.org/docs/Web/API/Server-sent_events/Using_server-sent_events#Event_stream_format)) based on the specified `Accept` header in the HTTP request.
+
+To start streaming log from the target process, run the following command:
+
+In PowerShell,
+
+```powershell
+PS> Start-Process http://localhost:52323/logs
+```
+
+In bash,
+```bash
+$ curl -H "Accept:application/x-ndjson" http://localhost:52323/logs --no-buffer
+```
+
```

Comment:
Perhaps clarify what logs are captured or provide some example logs from the asp.net helloworld?

Reply:
Added